### ISSUE TITLE
Adobe RGB JPEG opens as sRGB - detect input profile based on ColorSpace or InteropIndex tags

### ISSUE THREAD

**Issue by Beep6581** [Beep6581/RawTherapee#1805]
Originally reported on Google Code with ID 1821

```
For easier detection of the issue, take input image with saturated reds.
Open input JPG that is AdobeRGB profile.
Reds become dull. 
It is a similar effect, if image was treated as if it was in sRGB profile.

This does not apply to input tif-8 and tif-16.

```

Reported by `michaelezra000` on 2013-03-31 21:19:23

**Comment by Beep6581** [comment]
Stale.

**Comment by mantusma** [comment]
Hello, I have the same problem. Rawtherapee 5.7 don´t load the included color profile. Gimp loads the image correctly with the Adobe RGB profile.

As a workaround I exported the gimp profile "Compatible with Adobe RGB (1998).icc" and can load it manually in Rawtherapee.


![Uploading SAM4173.JPG…]()

**Comment by mantusma** [comment]
attached file

![SAM4173](https://user-images.githubusercontent.com/60468603/73444556-bf7b3e00-4393-11ea-8038-fc1c261d61be.JPG)

**Comment by Beep6581** [comment]
@mantusma your image does not include an ICC profile, but it does include relevant metadata:
```text
-JPEG-APP1-IFD0-ExifIFD:ColorSpace=Adobe RGB
-JPEG-APP1-IFD0-ExifIFD-InteropIFD:InteropIndex=R03 - DCF option file (Adobe RGB)
-JPEG-APP1-IFD0:PrimaryChromaticities=0.64 0.33 0.21 0.71 0.15 0.06
```

**Comment by Beep6581** [comment]
> As a workaround I exported the gimp profile "Compatible with Adobe RGB (1998).icc" and can load it manually in Rawtherapee.

You don't need to do that. Just open the JPEG in RT, set the input profile to "Custom" and select "RTv4_Medium.icc" which is equivalent to AdobeRGB, or select some other AdobeRGB.icc from disk.

**Comment by mantusma** [comment]
Thank you for your quick response! I tested the "RTv4_Medium.icc" in comparison to the Gimp profile. It is very similar but not identical but a much better "fallback" solution than the sRGB profile.

If I open the JPG with the "RTv4_Medium.icc" or "Compatible with Adobe RGB (1998).icc" file, should I save it with the "RTv4_Medium.icc" or "RTv4_srgb.icc"profile? The File size of the resulting JPG from RT are the identical and the colors also look identical in my opinion.

**Comment by Beep6581** [comment]
@mantusma save as you normally would - typically sRGB (RTv4_sRGB.icc).

**Comment by ValZapod** [comment]
> Maybe from WhitePoint/Primary Chromaticities/Y
> Cb Cr coefficients.

Yes, obiously. Same happen with my Nikon D5100 (you need to select Adobe RGB in menu).
![_DSC1521](https://user-images.githubusercontent.com/31514790/190709656-f97c1ce6-860d-4b9b-afc0-dac336b7ad25.JPG)



> tested the "RTv4_Medium.icc" in comparison to the Gimp profile

You really should use AdobeRGB1998.icc profile -- official.

What is worse is that .NEF that does have metadata about Adobe RGB too is no way to apply.

**Comment by ValZapod** [comment]
@Beep6581 Did you see my photo? I can also give a RAW, same problem, so much for RAW not having color profiles! Would be nice to fix, since you already have a way to plug in primaries values directly. Of course compared to ICC those values are not chromatically adapted to D50. Those should be used already as is.

**Comment by Beep6581** [comment]
This issue is about RT reading the color profile of JPEG images. That issues seems solved, 5.9 seems to read ICC profiles embedded in JPEG images correctly, so closing.

**Comment by ValZapod** [comment]
@Beep6581 There are no ICC profiles in this images. It is metadata. And it still does not work last time I checked the nightly.
